<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab4OnOff</class>
 <widget class="QWidget" name="Tab4OnOff">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pPBLamp1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="IotQtClient.qrc">
     <normaloff>:/IotQtClient/img/light_off.png</normaloff>
     <normalon>:/IotQtClient/img/light_on.png</normalon>:/IotQtClient/img/light_off.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>130</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pPBSpot2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>75</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>pPBSpot2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pPBOne">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>pPBOne</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pPBSpot1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>75</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>pPBSpot1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pPBLamp2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="IotQtClient.qrc">
     <normaloff>:/IotQtClient/img/light_off.png</normaloff>
     <normalon>:/IotQtClient/img/light_on.png</normalon>:/IotQtClient/img/light_off.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pPBTwo">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>pPBTwo</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="IotQtClient.qrc"/>
 </resources>
 <connections/>
</ui>
